airbyte-integrations/bases/source-acceptance-test/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.1.19
+Check record structure before validating it against json schema.
```

Comment:
I think this (and the PR title) should be a little clearer about what's actually changing here..."check record structure" is vague. Technically we are asserting a non-empty overlap between the fields present in the record and the schema because schema checks can be insufficient when there is zero overlap between data in the record and the schema (since additional properties = true in many cases)